modules/lily-compiler/src/main/java/io/github/tomboyo/lily/compiler/ast/AstReference.java

```diff
@@ -3,5 +3,12 @@
 import java.util.List;
 
 public record AstReference(
```

Comment:
AstReference may be redundant with Fqn, SimpleName, and PackageName. We should explore collapsing these together in a separate PR.